cmd/destroy/destroy.go

```diff
@@ -237,11 +238,48 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 	} else {
 		manifest, err := model.GetManifestV2(opts.ManifestPath)
 		if err != nil {
-			// Log error message but application can still be deleted
+			// error will only be available when there is no valid manifest found
+			// hence, we need not to destroy because the manifest would never be deployed
+			// Return the error message
 			oktetoLog.Infof("could not find manifest file to be executed: %s", err)
 			manifest = &model.Manifest{
 				Destroy: &model.DestroyInfo{},
 			}
+			return nil, err
+		}
+
+		// add the commands for the destroy section depending upon which type of manifest it is
+		if manifest.Destroy == nil {
+			if manifest.Type == model.ChartType {
+				chartPath, _ := discovery.GetHelmChartPath(opts.ManifestPath)
+				chartPath, _ = filepath.Rel(opts.ManifestPath, chartPath)
```

Comment:
```suggestion
				chartPath, _ = filepath.Rel(opts.ManifestPath, chartPath)
```
The reason we're not checking the error here is that these errors would have already got detected while trying to deploy the application [here](https://github.com/okteto/okteto/blob/master/pkg/model/manifest.go#L518).  _The application won't be destroyed if it has not been deployed on Okteto cloud initially. Hence a user needs to deploy the application first, if the error is found it'll automatically be returned and no destroy section will be executed. When the context gets over to destroy section after being deployed, there will be no errors expected._